docs/index.html

```diff
@@ -177,6 +184,8 @@ <h1 id="iban-info-title" class="text-2xl font-bold text-primary mb-4">IBAN Infor
             document.querySelector('label[for="name"]').textContent = t.nameLabel;
             document.querySelector('label[for="details"]').textContent = t.detailsLabel;
             document.getElementById('iban-info-desc').textContent = t.formDesc;
+            // Update disclaimer text
```

Comment:
Remove this superfluous and inconsistent code comment (the other updates don't have (need) comments like this one)